lib/components/src/controls/Boolean.stories.tsx

```diff
@@ -11,11 +11,18 @@ const Template = (initialValue?: boolean) => {
   return (
     <>
       <BooleanControl name="boolean" value={value} onChange={(newVal) => setValue(newVal)} />
-      <p>value: {typeof value === 'boolean' ? value.toString() : value}</p>
+      <pre>{JSON.stringify(value) || 'undefined'}</pre>
```

Comment:
`String(value)` would've done the trick. Unless you want additional quotes around strings.

Reply:
This is just common to all the stories for the different controls